src/components/Navigation/style.css

```diff
@@ -68,3 +68,45 @@
 .Navigation--Mobile {
   height: 101px;
 }
+.Mobile{
```

Comment:
Please use the `Navigation--` prefix, when defining new classes, so no classes in other components are affected.

Reply:
Added Prefix to its Navigation classes